integrated_channels/sap_success_factors/utils.py

```diff
@@ -266,3 +320,13 @@ def transform_language_code(code):
     language_family = SUCCESSFACTORS_OCN_LANGUAGE_CODES[language_code]
     language_name = language_family.get(country_code, language_family['_'])
     return language_name
+
+
+def safe_extract_key(data, key, default=''):
```

Comment:
I'd start with https://github.com/edx/edx-enterprise/blob/master/enterprise/utils.py -- at some point we might review all of the generalizations located in that module and see about lifting them to another library, but for now utils.py makes sense to me.